app/models/concerns/article.rb

```diff
@@ -84,7 +84,7 @@ def send_to_alegre(id)
       obj = self.find_by_id(id)
       return if obj.project_media.nil?
       obj.text_fields.each do |field|
-        ::Bot::Alegre.send_field_to_similarity_index(obj.project_media, field)
+        ::Bot::Alegre.relate_project_media_async(obj.project_media, field)
```

Comment:
This affects fact-checks actually, so we don't need to change it to the scope of this ticket. Let's focus only on Explainers first.